package.json

```diff
@@ -48,7 +48,7 @@
     "htmlparser2": "^6.0.0",
     "husky": "^4.3.8",
     "lint-staged": "^10.5.3",
-    "npm-merge-driver": "^2.3.6",
+    "npm-merge-driver": "^0.0.0",
```

Comment:
Looks like something went wrong here

Reply:
Yeah I found this curious as well. Thanks for catching it :)

Reply:
Fixed in 1b9394d00b5f31a2fd4ef7d48dbd7d164d1f20a8